packages/ui/components/form-core/types/FocusMixinTypes.ts

```diff
@@ -25,6 +25,12 @@ export declare class FocusHost {
    */
   blur(): void;
 
+  /**
+  * Synchronizes property values when attributes change.
+  * @category attributes
+  */
+    attributeChangedCallback(name: string, _old: string | null, value: string | null): void;
```

Comment:
Was TS complaining about it? I expect this to be already available on HTMLElement, which LItElement inherits from